transcendence-app/src/user/dto/user.dto.ts

```diff
@@ -7,7 +8,7 @@ import {
   ValidateIf,
 } from 'class-validator';
 
-export class UserDto {
+export class UserRequestDto {
```

Comment:
I like to differentiate between in and out DTOs so we don't mix up them so that's why I was adding the `Request` thing. I will update it to `CreateUserRequestDto` ;)